lib/ansible/modules/network/avi/avi_cloud.py

```diff
@@ -176,31 +211,39 @@
     tenant_ref:
         description:
             - It is a reference to an object of type tenant.
+        type: str
     url:
         description:
             - Avi controller URL of the object.
+        type: str
     uuid:
         description:
             - Unique object identifier of the object.
+        type: str
     vca_configuration:
         description:
             - Vcloudairconfiguration settings for cloud.
+        type: dict
     vcenter_configuration:
         description:
             - Vcenterconfiguration settings for cloud.
+        type: dict
     vtype:
         description:
             - Cloud type.
             - Enum options - CLOUD_NONE, CLOUD_VCENTER, CLOUD_OPENSTACK, CLOUD_AWS, CLOUD_VCA, CLOUD_APIC, CLOUD_MESOS, CLOUD_LINUXSERVER, CLOUD_DOCKER_UCP,
             - CLOUD_RANCHER, CLOUD_OSHIFT_K8S, CLOUD_AZURE, CLOUD_GCP.
             - Default value when not specified in API or module is interpreted by Avi Controller as CLOUD_NONE.
         required: true
+        type: str
+
+
 extends_documentation_fragment:
     - avi
 '''
 
 EXAMPLES = """
-  - name: Create a VMware cloud with write access mode
+  - name: Create a VMWare cloud with write access mode
```

Comment:
Lower-case `w` is correct. See https://www.vmware.com/. 
 ```suggestion
  - name: Create a VMware cloud with write access mode
```